modules/assim.sequential/R/sda.enkf.R

```diff
@@ -75,18 +77,25 @@ sda.enkf <- function(settings, obs.mean, obs.cov, IC = NULL, Q = NULL, adjustmen
     sampleIDs <- c(1:n.inputs,sample.int(n.inputs, (nens - n.inputs), replace = TRUE))
   }
   
-  ens.inputs <- list()
+  source('~/pecan/modules/assim.sequential/R/met_filtering_helpers.R')
```

Comment:
the call to source is a code development hack that shouldn't be necessary -- the met_filtering_helpers.R content will load as part of the package. It's also not stable since some people don't install pecan at ~/pecan (e.g. on my laptop I have pecan at ~/git/pecan)